    Source: http://code.google.com/p/ffmpegthumbnailer/issues/detail?id=97
    Upstream: Bug #97
    Reason: compilation error with current ffmpeg version
--- ffmpegthumbnailer-2.0.7/libffmpegthumbnailer/moviedecoder.cpp-orig	2011-07-24 11:00:59.000000000 +0100
+++ ffmpegthumbnailer-2.0.7/libffmpegthumbnailer/moviedecoder.cpp	2012-06-14 16:11:09.853106991 +0100
@@ -58,13 +58,12 @@
 void MovieDecoder::initialize(const string& filename)
 {
     av_register_all();
-    avcodec_init();
     avcodec_register_all();

     string inputFile = filename == "-" ? "pipe:" : filename;
     m_AllowSeek = (filename != "-") && (filename.find("rtsp://") != 0);

-    if ((!m_FormatContextWasGiven) && av_open_input_file(&m_pFormatContext, inputFile.c_str(), NULL, 0, NULL) != 0)
+    if ((!m_FormatContextWasGiven) && avformat_open_input(&m_pFormatContext, inputFile.c_str(), NULL, NULL) != 0)
     {
 		destroy();
         throw logic_error(string("Could not open input file: ") + filename);
